src/spotify.ts

```diff
@@ -0,0 +1,139 @@
+const completionSpec: Fig.Spec = {
+  name: "spotify",
+  description: "CLI to use Spotify from the terminal",
+  subcommands: [
+    {
+      name: "play",
+      description: "Resumes playback where Spotify last left off",
+      args: {
+        name: "song name",
+        description: "Finds a song by name and plays it",
+        isOptional: true,
+      },
+      subcommands: [
+        {
+          name: "album",
+          description: "Finds an album by name and plays it",
+          args: {
+            name: "album name",
+            description: "The album name you want to play",
+            isOptional: false,
+          },
+        },
+        {
+          name: "artist",
+          description: "Finds an artist by name and plays it",
+          args: {
+            name: "artist name",
+            description: "The artist name you want to play",
+            isOptional: false,
+          },
+        },
+        {
+          name: "list",
+          description: "Finds a playlist by name and plays it",
+          args: {
+            name: "playlist name",
+            description: "The playlist name you want to play",
+            isOptional: false,
+          },
+        },
+        {
+          name: "uri",
+          description: "Plau songs from specific uri",
+          args: {
+            name: "uri",
+            description: "The uri of the song you want to play",
+            isOptional: false,
+          },
+        },
+      ],
+    },
+    {
+      name: "next",
+      description: "Skips to the next song in a playlist",
+    },
+    {
+      name: "prev",
+      description: "Returns to the previous song in a playlist",
+    },
+    {
+      name: "replay",
+      description: "Replays the current track from the beginning",
+    },
+    {
+      name: "pos",
+      description: "Jumps to a time (in secs) in the current song",
+      args: {
+        name: "time",
+        description: "Exact time in secs to jump in",
+        isOptional: false,
+      },
+    },
+    {
+      name: "pause",
+      description: "Pauses (or resumes) Spotify playback",
+    },
+    {
+      name: "stop",
+      description: "Stops playback",
+    },
+    {
+      name: "quit",
+      description: "Stops playback and quits Spotify",
+    },
+    {
+      name: "vol",
+      description: "Shows the current Spotify volume",
+
+      args: {
+        name: "amount",
+        description: "Sets the volume to an amount between 0 and 100",
+        isOptional: true,
+      },
+      subcommands: [
+        { name: "up", description: "Increases the volume by 10%" },
+        { name: "down", description: "Decreases the volume by 10%" },
+      ],
```

Comment:
This should be condensed to the following structure:
```suggestion
      args: {
        name: "amount",
        description: "Sets the volume to an amount between 0 and 100",
        isOptional: true,
        suggestions: [
          { name: "up", description: "Increases the volume by 10%" },
          { name: "down", description: "Decreases the volume by 10%" },
        ],
      },
```